docs: Search Metadata Batch 02 (#14589)
hashicorp/consul: website/content/docs/security/security-models/nia.mdx

---
layout: docs
page_title: Security Models: Network Infrastructure Automation (NIA)
description: >-

> The NIA security model details requirements and recommendations for securing access to your network when using Consul-Terraform-Sync (CTS). Learn about potential threats and how to protect Consul from malicious actors.

Reviewer: ```suggestion
The NIA security model details requirements and recommendations for securing your deployment of Consul-Terraform-Sync (CTS). Learn about potential threats and how to protect CTS from malicious actors.
```
